setup.py

```diff
@@ -103,11 +100,13 @@ def is_platform_mac():
             include_dirs=['pyuvdata/uvdata/src']
         )
     ],
-    'scripts': glob.glob('scripts/*'),
-    'version': version.version,
+    'scripts': [fl for fl in glob.glob('scripts/*') if not os.path.isdir(fl)],
+    'use_scm_version': {
+        'local_scheme': branch_scheme
+    },
     'include_package_data': True,
-    'setup_requires': ['pytest-runner', 'numpy>=1.15'],
-    'install_requires': ['numpy>=1.15', 'scipy', 'astropy>=2.0', 'h5py'],
+    # 'setup_requires': ['pytest-runner', 'numpy>=1.15'],
+    'install_requires': ['numpy>=1.15', 'six>=1.10', 'scipy', 'astropy>=2.0', 'h5py'],
```

Comment:
Why did `six` get re-introduced here?

Reply:
almost certainly because I botched the rebase/merge